### PR TITLE
Add number of TPC clusters used for PID to AO2D

### PR DESCRIPTION
**Pull request by mpuccio**
This is an alternative implementation to #13417, to be discussed at the next WP14 which one (if any) will go in production

### CODE REVIEW

**Comment by mpuccio** [comment]
@pzhristov on Monday we can merge this (better not just before the weekend)

File: Framework/Core/include/Framework/AnalysisDataModel.h [hunk 1]
```diff
@@ -600,16 +603,47 @@ DECLARE_SOA_TABLE_FULL_VERSIONED(StoredTracksExtra_001, "TracksExtra", "AOD", "T
...
                            track::v001::DetectorMap);
+DECLARE_SOA_EXTENDED_TABLE(TracksExtra_002, StoredTracksExtra_002, "TRACKEXTRA", //! Additional track information (clusters, PID, etc.)
+                           track::v001::DetectorMap);
 
 DECLARE_SOA_TABLE(Run2TrackExtras, "AOD", "RUN2TRACKEXTRA",
                   track::ITSSignal);
 
-using StoredTracksExtra = StoredTracksExtra_001;
-using TracksExtra = TracksExtra_001;
+using StoredTracksExtra = StoredTracksExtra_002;
```
Comment (jgrosseo) [inline Framework/Core/include/Framework/AnalysisDataModel.h]: Hi @mpuccio 
Shouldn't we first merge it with the new table but keeping 001 the default
Then create the converter
Then change the default from 001 to 002?

Comment (mpuccio) [reply Framework/Core/include/Framework/AnalysisDataModel.h]: Yes! To do that I now pushed a first commit that does not modify the AODProducer. Last week also @f3sch pointed out this issue.

**Comment by mpuccio** [comment]
@pzhristov we are ready to merge (note that GH had issues today at some point, so the failures of the actions are not related). The current PR does not disturb O2Physics, but will enable us to commit the converters

**Review by pzhristov — APPROVED** [review]

**Pull request merged**